### PR TITLE
Intergrate Swappy to edit screenshots

### PR DESCRIPTION
**Pull request by arran-nz**
Integrates [Swappy](https://github.com/jtheoof/swappy) to edit the captured screenshots

### CODE REVIEW

**Comment by moverest** [comment]
Nice, I like it, thank you! I'd just make it optional by checking if Swappy is installed (and not ask if it's not). :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ If you are using Archlinux, you can install sway-interactive-screenshot with the
 - `rofi` to prompt what you want to take a screenshot of
 - `grim` to take the screenshot
 - `slurp` to select an area on the screen
+- `swappy` to edit the captured screenshot - https://github.com/jtheoof/swappy
```
Comment (moverest) [inline README.md]:
```suggestion
- [`swappy`](https://github.com/jtheoof/swappy) (optional) to edit the captured screenshot
```

**Comment by arran-nz** [comment]
Great idea, done :)

**Comment by moverest** [comment]
Thanks, [new release](https://github.com/moverest/sway-interactive-screenshot/releases/tag/1.1.0) is up.

**Pull request merged**